docs/references/global-shell.md

```diff
@@ -0,0 +1,153 @@
+---
+id: global-shell
+title: Global Shell
+---
+
+The Global App Shell is a new feature in DHIS2 v42. It’s a shared launching point for all of the apps: when you visit DHIS2, the Global Shell will open, and then the Global Shell will load apps inside of itself.
```

Comment:
I take it it's on purpose you call it "App" here, not in the other places ... "Global App Shell" vs "Global Shell"

Reply:
Yeah, I considered this a bit... I figured I would call it "App Shell" here to say what it's a shell of, but then "Global Shell" for short from then on

Reply:
"Global app shell" is also used in other comms about new features this version